fix: content overflow & scrolling issues

maiertech opened the pull request:
# Description of change

I fixed a number of scrolling issues.

## Links to any relevant issues

- #393
- Previously merged into dirty v0.7 in #398.

## How the change has been tested

How to test as a reviewer:

- Check out fixed routes in #393.
- For each route, launch UniMe on your desktop and make height of window small enough to trigger scrolling.
- When you scroll all the way down, content should not be covered by the bottom navigation.
- For the prompt routes use these sites for testing:
  - Test offers and sharing credentials: https://ministerie-van-defensie.dev.impierce.com/ (and subsequent steps),
  - Test connection: https://decathlon.co.nl/ or another web shop,
  - Scan QR-codes from screenshots: https://scanqr.org/.

## Definition of Done checklist

Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

maiertech committed:
Fix scrolling issues

Define prop `class` for `TopNavBar` to make it more explicit

Remove unneded class

Fix `/me/settings`

Fix `/me/settings/app`

Fix `/me/settings/about`

Fix `/me/settings/app/did`

Fix `/me/settings/app/keys`

Fix `/me/settings/app/language`

Fix `/me/settings/app/theme`

Fix `/me/settings/app/trust-list`

Fix `/me/settings/profile`

Fix `/prompt/accept-connection`

Fix `/prompt/credential-offer`

Fix `/prompt/share-credentials`

Fix layout issues by creating a new stacking context

M-Adam-Hus commented:
![](https://github.trello.services/images/mini-trello-icon.png) [fix: Fix scrolling issues](https://trello.com/c/CUNViQ8P/69-fix-fix-scrolling-issues)

Oran-Dan committed:
fix: support `StatusList2021Entry` credentials, fetch logo from `.well-known/openid-credential-issuer` (#414)

* WIP

* Add Thuiswinkel verification checkmark

* fix: fix logo_url binding, renameto thuiswinkel_validation

* Rename `thuiswinkel_waarborg_verification` to `thuiswinkel_verification` in frontend

* feat: add `issuance_date` to `ValidationResult`

* Change label for domain verification

* fix: fix `issuance_date`

* Add issuance date and change logo position

* chore(deps-dev): bump prettier-plugin-tailwindcss from 0.6.1 to 0.6.5 (#259)

Bumps [prettier-plugin-tailwindcss](https://github.com/tailwindlabs/prettier-plugin-tailwindcss) from 0.6.1 to 0.6.5.
- [Release notes](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/releases)
- [Changelog](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/compare/v0.6.1...v0.6.5)

---
updated-dependencies:
- dependency-name: prettier-plugin-tailwindcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* chore(deps-dev): bump @ianvs/prettier-plugin-sort-imports (#260)

Bumps [@ianvs/prettier-plugin-sort-imports](https://github.com/ianvs/prettier-plugin-sort-imports) from 4.2.1 to 4.3.0.
- [Release notes](https://github.com/ianvs/prettier-plugin-sort-imports/releases)
- [Changelog](https://github.com/IanVS/prettier-plugin-sort-imports/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ianvs/prettier-plugin-sort-imports/compare/v4.2.1...v4.3.0)

---
updated-dependencies:
- dependency-name: "@ianvs/prettier-plugin-sort-imports"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* chore(deps-dev): bump prettier-plugin-svelte from 3.2.3 to 3.2.5 (#262)

Bumps [prettier-plugin-svelte](https://github.com/sveltejs/prettier-plugin-svelte) from 3.2.3 to 3.2.5.
- [Changelog](https://github.com/sveltejs/prettier-plugin-svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/prettier-plugin-svelte/compare/v3.2.3...v3.2.5)

---
updated-dependencies:
- dependency-name: prettier-plugin-svelte
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Nander Stabel <nander.stabel@impierce.com>

* chore(deps-dev): bump prettier from 3.3.0 to 3.3.2 (#261)

Bumps [prettier](https://github.com/prettier/prettier) from 3.3.0 to 3.3.2.
- [Release notes](https://github.com/prettier/prettier/releases)
- [Changelog](https://github.com/prettier/prettier/blob/main/CHANGELOG.md)
- [Commits](https://github.com/prettier/prettier/compare/3.3.0...3.3.2)

---
updated-dependencies:
- dependency-name: prettier
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* Make `imageId` reactive

* fix: fix lint errors

* add: backend actions-reducers for editable trust list

* cargo fmt

* add: testing

* feat: trusted domains list (first version) (#352)

* ci: release version 0.6.8 (#346)

* fix: bindings

* feat: add default_trust_list.json

* test: add testing, not final

* feat: enable multiple trust lists

* chore: remove comment

* refactore: rename actions and reducers

* feat: add TrustList level actions

* refactor: trustlist owned bool, name to id, reset to default

* chore: fix domain name

* feat: add trust list management frontend

* refactor: implement lists view & domain list

* refactor: Wire up Tauri listeners in `onMount` of root layout (#322)

* Wire up Tauri state-changed listener in `onMount` of root layout

* Wire up Tauri error listener in root layout

* Update comments

* build: Replace `npm` with `pnpm` (#348)

* Swap out `npm` for `pnpm`
Fixes #347

* Address review comments

* chore: cargo fmt and fix binding naming

* feat: prevent edits on imported lists

* refactor: remove unused reset functionality

* feat: add placeholder when no lists

* refactor: trustlist actions

* refactor: remove trustlist default impl

* refactor: use updated Action names

* fix: rename broken bindings

* refactor: remove `Default` trait

* refactor: show `display_name` instead of `id`

* feat: update default trusted domains

---------

Co-authored-by: Daniel Mader <daniel.mader@impierce.com>
Co-authored-by: Thilo Maier <thilo.maier@impierce.com>

* feat: add `validate_linked_verifiable_presentations`

* refactor: apply clippy suggestions

* feat: use `ServiceEndpoint::from`

* refactor: remove unused variables

* refactor: replace icon imports

* feat: add alignment & criteria display

* chore: pnpm format

* chore: eslint failed

* feat: filter linked VPs based on trust list

* feat: display trusted domains as trusted issuers

* chore: try to fix git workflow build fail

* chore: try to fix git workflow build fail

* chore: add comment to ubuntu version fix

* chore: update serde_json dependency, make error messages static for consistency between version updates

* chore: cargo fmt

* chore: cargo clippy

* chore: fix tests

* chore: cargo fmt

* feat: add achievementType to badge overview display

* ci: release version 0.6.13

* fix: disable prerender trust-list

* WIP

* WIP

* WIP

* ci: release version 0.6.14

* chore: rename variable

* chore: fix log message

* chore: fix trusted domain validation

* feat: add default trust lists to all profiles at creation

* feat: add did:web url validation against trust_list, format

* chore: cargo clippy:

* feat: add fetching logo from well known endpoint

* chore: fix fallback icon, add comments

* chore: add credenco to trusted issuers

* ci: release version 0.6.15

* fix: set SD-JWT format check back to `VcSdJwt` (#394)

* feat: display `alignment` & `criteria` for Open Badges 3.0 (#386)

* ci: release version 0.6.12 (#370)

* chore(deps-dev): bump globals from 15.9.0 to 15.11.0 (#389)

Bumps [globals](https://github.com/sindresorhus/globals) from 15.9.0 to 15.11.0.
- [Release notes](https://github.com/sindresorhus/globals/releases)
- [Commits](https://github.com/sindresorhus/globals/compare/v15.9.0...v15.11.0)

---
updated-dependencies:
- dependency-name: globals
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* chore(deps-dev): bump typescript-eslint from 8.2.0 to 8.12.0 (#391)

Bumps [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/typescript-eslint) from 8.2.0 to 8.12.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/typescript-eslint/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v8.12.0/packages/typescript-eslint)

---
updated-dependencies:
- dependency-name: typescript-eslint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* chore(deps-dev): bump prettier-plugin-svelte from 3.2.6 to 3.2.7 (#382)

Bumps [prettier-plugin-svelte](https://github.com/sveltejs/prettier-plugin-svelte) from 3.2.6 to 3.2.7.
- [Changelog](https://github.com/sveltejs/prettier-plugin-svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/prettier-plugin-svelte/compare/v3.2.6...v3.2.7)

---
updated-dependencies:
- dependency-name: prettier-plugin-svelte
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* chore(deps-dev): bump postcss from 8.4.45 to 8.4.47 (#375)

Bumps [postcss](https://github.com/postcss/postcss) from 8.4.45 to 8.4.47.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.45...8.4.47)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* chore: resolve PR comments

* chore: resole PR comment

* chore: resolve PR comment

---------

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: Daniel Mader <daniel.mader@impierce.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* fix: use hash-based image identifier to prevent caching, use colored background for icon fallback (#396)

* fix: use colored background when icon fallback is displayed

* fix: use hash-based image identifier to prevent caching

* ci: release version 0.7.0

* fix: trusted issuer domain url validation (#397)

* fix: url validation for trusted issuers

* chore: replace expect() with ok()

* Make it more clear how scrolling is done

* Define prop `class` for `TopNavBar` to make it more explicit

* Remove unneded class

* Fix `/me/settings`

* Fix `/me/settings/app`

* Fix `/me/settings/about`

* Fix `/me/settings/app/did`

* Fix `/me/settings/app/keys`

* Fix `/me/settings/app/language`

* Fix `/me/settings/app/theme`

* Fix `/me/settings/app/trust-list`

* Fix `/me/settings/profile`

* Fix `/prompt/accept-connection`

* Fix `/prompt/credential-offer`

* Fix `/prompt/share-credentials`

* fix: adjust UniMe text color in dark mode

* feat: add terms & conditions

* fix: checkbox tick color in dark mode

* feat: update app icon

* ci: release version 0.7.1

* feat: update app icon

* feat: fix app icon

* fix: higher resolution app icon

* refactor: remove `Beta` from `productName`

* ci: release version 0.7.2

* refactor: update camera usage description

* refactor: update camera usage description

* Fix layout issues by creating a new stacking context (#410)

* ci: release version 0.7.3

* fix: support StatusList2021Entry, add logo fetching from well-known/openid-credential-issuer Credential Configurations Supported

* chore: refactor fn get_logo_uri

* chore: add domain to default trust list

* chore: cargo clippy

* chore: merge dev unime/ folder into branch

* chore: checkout dev identity-wallet/src/state/trustlist/

* chore: checkout dev into identity-wallet/ files

* chore: remove outdated files

* chore: remove outdated unime/ files

* chore: fix tests

* chore: remove outdated domain

* chore: add TODO in comment

* chore: fix PascalCasing

* chore: clarify log message

* chore: add spec in comment to alternative property name

* chore: remove workaround which should be fixed in frontend, add comments

* chore: fix comments

* chore: fix default trust list entries check in linked vp flow

* chore: fix PR comments

* chore
: improve log messages

* chore: update tests to include logo fetching through wiremock

---------

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: Nander Stabel <nander.stabel@impierce.com>
Co-authored-by: Thilo Maier <thilo.maier@impierce.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Daniel Mader <daniel.mader@impierce.com>

maiertech committed:
Fix one closing `div` too many

maiertech committed:
Merge remote-tracking branch 'origin/dev' into maiertech/issue393
